src/components/common/blocks/user-address-stats/style.js

```diff
@@ -31,32 +32,28 @@ export const Item = styled.div`
   ${media.mobile`
     border-right: none;
     margin-bottom: 2rem;
-    padding: 0;
+    padding: 0 0 1rem 0;
     
     &:last-child {
       margin-bottom: 0;
     }
   `};
 `;
 
-export const Label = styled.div`
-  text-align: left;
-  font-family: 'Futura PT Heavy', sans-serif;
-`;
-
 export const Data = styled.div`
-  margin-top: 1rem;
   display: flex;
+  text-transform: none;
   justify-content: flex-start;
   align-items: flex-end;
-  font-family: 'Futura PT Heavy';
-  font-size: 3.8rem;
+  font-family: 'Futura PT Medium';
+  font-size: 1.8em;
```

Comment:
Let's use the rem as standard CSS unit so the value will always be relative to the root html.

Reply:
got it :+1: